projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -75,6 +75,16 @@ public function __construct() {
 	 * @param Jetpack $jetpack object.
 	 */
 	public function on_jetpack_loaded( $jetpack ) {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended		
```

Comment:
Maybe move to https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php#L52

Reply:
good call